.github/workflows/ci.yml

```diff
@@ -253,3 +280,244 @@ jobs:
           git push -u origin $env:GH_DOC_BRANCH --force
         env:
           GH_DOC_BRANCH: gh-pages
+
+  build_test_package_others:
+    name: Build Test and Package for release
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019]
+        python-version: ['3.7', '3.9']
+
+    env:
+      ANSYS_VERSION: 221
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.1.4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - id: install-dpf
+        uses: pyansys/pydpf-actions/install-dpf-server@v1
+        with:
+          dpf-standalone-TOKEN: ${{secrets.DPF_PIPELINE}}
+          ANSYS_VERSION : ${{env.ANSYS_VERSION}}
+
+      - name: Set AWP_ROOT$env:ANSYS_VERSION
+        run: |
+          echo AWP_ROOT$env:ANSYS_VERSION
+          echo "AWP_ROOT$env:ANSYS_VERSION=${{ steps.install-dpf.outputs.SERVER }}"  | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+
+      - name: Install ansys-dpf-core
+        shell: cmd
+        run: |
+          pip install -r requirements_build.txt
+          python setup.py bdist_wheel
+          FOR /F %%a in ('dir /s/b dist\*.whl') do SET WHEELPATH=%%a
+          ECHO %WHEELPATH%
+          cd tests
+          pip install %WHEELPATH%
+
+      - name: Test import
```

Comment:
@PProfizi there is a lot of duplications between the 2 pipelines, could we not create new reusable actions in the actions repo instead?

Reply:
@cbellot000 Yes there is, and the next step is indeed to work on creating actions. I wanted to finish the pipelines for Linux and for PyDPF-Post before getting those together.